### ISSUE TITLE
Something is preventing multiple phone numbers from making it to the API

### ISSUE THREAD

**Issue by iandees** [RagtagOpen/phone-number-validator#11]
**Describe the bug**
See https://github.com/RagtagOpen/phone-number-validator/issues/4#issuecomment-502460807.

**To Reproduce**
1. Execute [this example API request](https://kce1bslyse.execute-api.us-east-1.amazonaws.com/dev/v1/number/validate?number=4158675309&number=2491858)

**Expected behavior**
Two results. One for the number `4158675309` and other for `2491858`.

A potential fix for this could be to switch to a different method of passing in multiple phone numbers. POSTing a JSON array of numbers? A single value with comma-separated values?

**Comment by bzupnick** [comment]
When I run it locally and hit [this endpoint](http://127.0.0.1:5000/v1/number/validate?number=4158675309&number=2491858) it returns a proper response of:
```

{
  "results": [
    {
      "formatted": {
        "e164": "+14158675309", 
        "national": "(415) 867-5309"
      }, 
      "input": "4158675309", 
      "location": {
        "description": "California", 
        "tz": [
          "America/Los_Angeles"
        ]
      }, 
      "reason": "possible", 
      "type": "fixed_or_mobile", 
      "valid": true
    }, 
    {
      "input": "2491858", 
      "reason": "too_short", 
      "valid": false
    }
  ]
}
```
Could it be a server configuration issue and not an application issue?

**Comment by iandees** [comment]
Yep! See [my comment in the other issue](https://github.com/RagtagOpen/phone-number-validator/issues/4#issuecomment-502460807). It's likely that some part of our serving stack doesn't support multiple values for the same query args, so we'd like to pick a better way of sending multiple numbers in a single request.
